### DISCUSSION TITLE
Error: The server does not support SSL connections

### DISCUSSION THREAD

**Discussion by diegodp-zz** [medusajs/medusa#5520] [Q&A] [answered]
hello, i am getting this error when trying to deploy in hostinger. 
 i use this 
 module.exports = {  
 projectConfig: {  
 database_extra:   
 process.env.NODE_ENV !== "development"  
 ? { ssl: { rejectUnauthorized: false } }  
 : {},  
 // ...  
 },  
 // ...  
}  

but the error persist

**Comment by diegodp-zz** [comment, marked as answer]
I find the problem, i needed to configure the postgres to be SSL on 

If someon have the same problem with me, i have followed these instructions and worked.
https://www.howtoforge.com/how-to-enable-ssl-for-postgresql-connections/

**Comment by fxck** [comment]
I think this is still a valid errors, something is not right, I have both `ssmode=disable` in `DATABASE_URL` as well as 

```typescript
    databaseDriverOptions: {
      connection: {
        ssl: {
          rejectUnauthorized: false
        }
      }
    },
```

in config, yet I'm still getting "The server does not support SSL connections"

**Comment by oklenum** [comment]
I had a typo in the connection string...
sslmode=disable -> ssl_mode=disable
Fixed it for me 🙃

**Reply by Ivaylo-Korakov** [reply]
If hosting on Coolify the DB is on an internal docker network and no SSL is configured by default when deploying Postgres.
You can either generate custom keys and adjust the deployment settings of postgress to use SSL or use the `?ssl_mode=disabled`.

**Reply by xcflyworld** [reply]
Thank you very much, it's done，use  this    ssl_mode=disabled

**Reply by milan-lakhani** [reply]
Yes! thanks. I wrote sslmode=disable from some stack overflow post and it didn't work, then I came across this! Thanks!

**Comment by fxck** [comment]
The correct param would be `sslmode` (https://www.postgresql.org/docs/current/libpq-ssl.html), but from what I found the problem was actually `rejectUnauthorized: false`, contrary to what the docs say, it should not be defined at all.

So if you want to disable SSL, do a combo of `ssl_mode=disable` + no `databaseDriverOptions` whatsoever.

**Comment by ddemeter56** [comment]
previously when I set NODE_ENV=production and sslmode=disable it was working for me.. Now I have the same error as you, and I changed nothing...
